app/views/EmploymentRelatedSecuritiesView.scala.html

```diff
@@ -21,7 +21,7 @@
     appConfig: AppConfig)
 
 @()(implicit request: Request[_], messages: play.api.i18n.Messages)
-
+non-UK-resident-landlords
```

Comment:
Does this need to be added?

Reply:
No, it doesn't, well spotted. I'll remove